pkg/console/dashboard_panels.go

```diff
@@ -392,16 +403,39 @@ func getHarvesterStatus() string {
 		return wrapColor(statusUnknown, colorYellow)
 	}
 
-	status, err := harvesterPodStatus()
+	harvesterRunning, harvesterReady, harvesterStatus, err := getPodStatus("harvester-system", "app.kubernetes.io/name=harvester")
```

Comment:
I don't think users can distinguish `running` and `ready` without knowledge of pod status.

Reply:
Ready -> green color
Running -> yellow color